hi-ml-histopathology/src/histopathology/scripts/generate_crossval_report.py

```diff
@@ -61,14 +61,37 @@ def generate_html_report(parent_run_id: str, output_dir: Path, workspace_config_
         test_metrics_table = get_crossval_metrics_table(metrics_df, test_metrics_list)
         report.add_tables([test_metrics_table])
 
-        # Add test ROC and PR curves
-        crossval_dfs = collect_crossval_outputs(parent_run_id, report_dir, aml_workspace, overwrite=overwrite)
+    report.add_heading("Model outputs", level=2)
+
+    has_val_and_test_outputs = crossval_runs_have_val_and_test_outputs(parent_run)
+
+    if has_val_and_test_outputs:
+        # Add val. ROC and PR curves
+        val_outputs_filename = "val/test_output.csv"
+        val_outputs_dfs = collect_crossval_outputs(parent_run_id, report_dir, aml_workspace,
+                                                   output_filename=val_outputs_filename,
+                                                   overwrite=overwrite)
 
-        report.add_heading("Test ROC and PR curves", level=2)
-        fig = plot_crossval_roc_and_pr_curves(crossval_dfs)
-        roc_pr_curves_fig_path = report_dir / "roc_pr_curves.png"
-        fig.savefig(roc_pr_curves_fig_path, bbox_inches='tight')
-        report.add_images([roc_pr_curves_fig_path], base64_encode=True)
+        report.add_heading("Validation ROC and PR curves", level=3)
+        fig = plot_crossval_roc_and_pr_curves(val_outputs_dfs, scores_column='prob_class1')
+        val_roc_pr_curves_fig_path = report_dir / "val_roc_pr_curves.png"
+        fig.savefig(val_roc_pr_curves_fig_path, bbox_inches='tight')
+        report.add_images([val_roc_pr_curves_fig_path], base64_encode=True)
+
+    if include_test:
```

Comment:
We already check `if include_test` on line 58 - can this block move inside there?

Reply:
As @fepegar said, the report's layout is currently:
* General run information
* AML metrics
  * Training curves
  * Validation metrics table
  * [optional] Test metrics table
* ROC/PR plots
  * Validation curves
  * [optional] Test curves

(Report page is fairly tall, so screenshot(s) would be awkward 😄)

Reply:
Not asking you use this, but I got curious and thought I'd share (maybe this should go to Pro tips? it's quite nice!):

![Web capture_1-4-2022_184731_techdows com](https://user-images.githubusercontent.com/12688084/161315805-a9f1f62e-65f9-4135-8905-e1b41060eac1.jpeg)

Reply:
@fepegar It works! 👀
![image](https://user-images.githubusercontent.com/7073707/161317431-418211e3-7ab5-451e-a0b8-9b53db20c54d.png)